pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -280,3 +288,30 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	return config, nil
 }
+
+// ensureDockerAPIVersionForParallelImagePulls returns true if the docker API version
+// is >=1.21, else false.
+func ensureDockerAPIVersionForParallelImagePulls(dh *docker.Helper, version string) bool {
```

Comment:
You can create the helper in `BuildKubernetesNodeConfig`, get the client, and pass the client here.